assets/training/model_management/src/run_model_preprocess.py

```diff
@@ -155,6 +157,7 @@ def run():
             logger.warning(f"{TRUST_CODE_KEY} is not provided for hf_config_args. Using {TRUST_CODE_KEY}.")
     preprocess_args["task"] = task_name.lower()
     preprocess_args["model_id"] = model_id if model_id else preprocess_args.get("model_id")
+    preprocess_args["model_flavor"] = model_flavor if model_flavor else "HFtransformersV2"
```

Comment:
Please define "HFtransformersV2" under constants. Basically we can add all flavors that is supported

Reply:
Will handle in a different PR